README.md

```diff
@@ -71,7 +71,7 @@ HISTORY_DB_ADDRESS          |History database's address
 HISTORY_DB_PORT             |History database's port                                       |27017
 HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
 LOG_LEVEL                   | Sets the log level                                           | "INFO"
-
+DOJOT_PERSIST_NOTIFICATIONS_ONLY                   |Defines a value to change the behavior of the service                                           |False
```

Comment:
This definition `Defines a value to change the behavior of the service`  doesn't make sense. Be clearer!!